### PR TITLE
pin docker version for local build target

### PR DESCRIPTION
**Pull request by michaelshobbs**
closes #296

### CODE REVIEW

File: Dockerfile.build [hunk 1]
```diff
@@ -1,2 +1,19 @@
...
+     curl \
+     gnupg2 \
+     software-properties-common
+RUN curl -fsSL https://download.docker.com/linux/$(. /etc/os-release; echo "$ID")/gpg | apt-key add -
+RUN add-apt-repository \
+   "deb [arch=amd64] https://download.docker.com/linux/$(. /etc/os-release; echo "$ID") \
+   $(lsb_release -cs) \
+   stable"
+RUN apt-get update -qq \
+ && apt-get install -qq -y --force-yes docker-ce=17.12.0~ce-0~debian
```
Comment (iSDP) [inline Dockerfile.build]: no need to suffix `~ce-0~debian` iirc

Comment (michaelshobbs) [reply Dockerfile.build]:
```
E: Version '17.12.0' for 'docker-ce' was not found
```

Comment (iSDP) [reply Dockerfile.build]: shite. the web lied.

Comment (michaelshobbs) [reply Dockerfile.build]: i generally test before i open PRs 😄

Comment (iSDP) [reply Dockerfile.build]: Windows is a turd when it comes to development

**Pull request merged**